lib/Memecoin.ts

```diff
@@ -1455,4 +1485,48 @@ export class MemecoinSDK {
         return UNISWAP_V3_ROUTER
     }
   }
+
+  private async getPoolLiquidityAtCurrentTick(poolAddress: EthAddress): Promise<bigint> {
```

Comment:
create a UniswapV3 class under `uniswapv3.ts` and move this function to it. basically whoever needs this, will have to do 

```this.uniswapv3.getPoolLiquidityAtCurrentTick(poolAddress)```